src/components/EReceiptThumbnail.tsx

```diff
@@ -37,30 +34,47 @@ const backgroundImages = {
     [CONST.ERECEIPT_COLORS.PINK]: eReceiptBGs.EReceiptBG_Pink,
 };
 
-function EReceiptThumbnail({transaction}) {
+function EReceiptThumbnail({transaction}: EReceiptThumbnailProps) {
     const styles = useThemeStyles();
     const StyleUtils = useStyleUtils();
-    // Get receipt colorway, or default to Yellow.
-    const {backgroundColor: primaryColor, color: secondaryColor} = StyleUtils.getEReceiptColorStyles(StyleUtils.getEReceiptColorCode(transaction));
 
     const [containerWidth, setContainerWidth] = useState(0);
     const [containerHeight, setContainerHeight] = useState(0);
 
-    const onContainerLayout = (event) => {
+    const backgroundImage = useMemo(() => {
+        if (!transaction) {
+            return;
+        }
```

Comment:
Please make sure this update doesn't cause regression. Also maybe consider updating `getEReceiptColorCode` typing

Reply:
Updated typings